app/views/layouts/application.html.erb

```diff
@@ -20,10 +43,20 @@
         <%= link_to "Edit User", edit_user_registration_path %>
                 
       <% end %>
+>>>>>>> c26401240e6082825e39df9c17ad34f9870412e7
```

Comment:
Looks like you have a merge conflict here

Reply:
Yeah I really messed something up there. Its fixed now.